app/src/main/java/io/github/droidkaigi/confsched2018/presentation/staff/StaffViewModel.kt

```diff
@@ -0,0 +1,48 @@
+package io.github.droidkaigi.confsched2018.presentation.staff
+
+import android.arch.lifecycle.Lifecycle
+import android.arch.lifecycle.LifecycleObserver
+import android.arch.lifecycle.LiveData
+import android.arch.lifecycle.OnLifecycleEvent
+import android.arch.lifecycle.ViewModel
+import io.github.droidkaigi.confsched2018.data.repository.StaffRepository
+import io.github.droidkaigi.confsched2018.model.Staff
+import io.github.droidkaigi.confsched2018.presentation.Result
+import io.github.droidkaigi.confsched2018.presentation.common.mapper.toResult
+import io.github.droidkaigi.confsched2018.util.defaultErrorHandler
+import io.github.droidkaigi.confsched2018.util.ext.toLiveData
+import io.github.droidkaigi.confsched2018.util.rx.SchedulerProvider
+import io.reactivex.disposables.CompositeDisposable
+import io.reactivex.rxkotlin.addTo
+import io.reactivex.rxkotlin.subscribeBy
+import javax.inject.Inject
+
+class StaffViewModel @Inject constructor(
+        private val repository: StaffRepository,
+        private val schedulerProvider: SchedulerProvider
+) : ViewModel(), LifecycleObserver {
+
+    private val compositeDisposable: CompositeDisposable = CompositeDisposable()
+
+    val staff: LiveData<Result<List<Staff>>> by lazy {
+        repository.staff
+                .toResult(schedulerProvider)
+                .toLiveData()
+    }
+
+    @OnLifecycleEvent(Lifecycle.Event.ON_CREATE)
+    fun onCreate() {
+        refreshStaff()
+    }
+
+    private fun refreshStaff() {
```

Comment:
Is this method need? 🤔

Reply:
This method was not necessary. I fixed in https://github.com/DroidKaigi/conference-app-2018/pull/403/commits/a3064f5d4838d0bd2e3ce76b05a3fc46099dbac5